src/test/java/com/iexec/blockchain/command/task/initialize/TaskInitializeTest.java

```diff
@@ -60,7 +58,7 @@ void shouldNotInitializeTaskSinceCannotOnChain() {
         String chainTaskId = taskInitializeService.start(CHAIN_DEAL_ID, TASK_INDEX);
 
         Assertions.assertTrue(chainTaskId.isEmpty());
-        verify(queueService, times(0)).addExecutionToQueue(any(), anyBoolean());;
+        verify(queueService, times(0)).addExecutionToQueue(any(), anyBoolean());
```

Comment:
```suggestion
        verify(queueService, never()).addExecutionToQueue(any(), anyBoolean());
```
Just a slight preference on my side

Reply:
I think it's better to stay with `times(0)` as it keeps the same structure as this line:
https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/blob/6c563a422d0922e341360df384cc69ef5efb2ded/src/test/java/com/iexec/blockchain/command/task/initialize/TaskInitializeTest.java#L49

Reply:
This second structure can be written
`verify(queueService).addExecutionToQueue(any(),eq(false))`

`times(1)` is a default value that can be omitted.
I tend to prefer to have an improved readability by removing `times(1)` and using `never()`.
But it's only a preference.

Reply:
That's a discussion we had with @zguesmi and he tends to prefer the other way which offers some consistency between each test. I now agree with him. So I'd prefer to stay as is :)